src/md/src/GettingStarted/Plugins.md

```diff
@@ -100,7 +100,7 @@ Details on how to configure a specific plugin please find in the [Integration se
 * [integration with SauceLabs](https://reportportal.io/docs/Integrations%3Esauce-labs-integration)
 * [integration with E-mail server](https://reportportal.io/docs/Integrations%3Eglobal-e-mail-server-integration)
 * [integration with LDAP](https://reportportal.io/docs/LDAP-Auth-integration)
-* [integration with Active directory](https://reportportal.io/docs/Active-Directory-Auth)
+* [integration with Active directory](https://reportportal.io/docs/Active-Directory-Auth-integration)
```

Comment:
Should we remove `integration` here too for AD?

Reply:
Sorry, it appears after git pull.